app/main/views/briefs.py

```diff
@@ -360,12 +358,7 @@ def view_response_result(brief_id):
         supplier_id=current_user.supplier_id
     )['briefResponses']
 
-    legacy_brief = True
```

Comment:
I probably wouldn't go to too much/any effort in trying to tidy up the bit around feature flags as it is about to get stripped out (apart from I think one location) as the old flow is no longer needed. There is currently a pull request up that does this - https://github.com/alphagov/digitalmarketplace-supplier-frontend/pull/635